src/main/java/org/tugraz/sysds/runtime/util/DataConverter.java

```diff
@@ -955,8 +977,8 @@ public static String toString(TensorBlock mb) {
 	 * @param decimal number of decimal places to print, -1 for default
 	 * @return tensor as a string
 	 */
-	public static String toString(TensorBlock tb, boolean sparse, String separator, String lineseparator,
-			String leftBorder, String rightBorder, int rowsToPrint, int colsToPrint, int decimal){
+	public static String toString(HomogTensor tb, boolean sparse, String separator, String lineseparator,
+	                              String leftBorder, String rightBorder, int rowsToPrint, int colsToPrint, int decimal){
```

Comment:
Please never use this Python alignment of arguments, additional lines are supposed to be intended via a single tab.

Reply:
For long method names + return arguments, this is unreadable and also violates our indentation policy of tabs over spaces.

Reply:
A single tab or two tabs? I saw both so I am not sure which one I should go for.